configure.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 source env.sh # source this from your run script too
-mkdir build || true # make sure not to commit this to svn or git
+mkdir -p build || true # make sure not to commit this to svn or git
```

Comment:
Do we need this?

Reply:
Yes. Otherwise make will return with an error if the directory already exists.

Reply:
Not really, but it squashes a warning "cannot make build already exists"

Reply:
With `-p` you could remove the `|| true`. From the man-page:
```
     -p      Create intermediate directories as required.  If this option is not specified, the full path prefix of each operand must already exist.  On the other hand, with this option specified, no error will be reported if a directory given as an
             operand already exists.  Intermediate directories are created with permission bits of rwxrwxrwx (0777) as modified by the current umask, plus write and search permission for the owner.
```

```suggestion
mkdir -p build
```

Reply:
> Not really, but it squashes a warning "cannot make build already exists"

Well, at least on juwels the second mkdir returns 1 instead of 0. So there is an error along with the message. We could remove the || true if we use -p, I think.

Reply:
```suggestion
mkdir -p  # make sure not to commit this to svn or git
```